Project-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,24 @@
 import React from 'react';
 import './Button.css';
+
+const ActionButton = props => {
+    return (
+        <div className={`ActionButton ${props.buttonStyle}`}>
```

Comment:
See how powerful passing down props are? We have a custom buttonstyle we are passing down to the class.